core/App/components/misc/CredentialCard.tsx

```diff
@@ -124,67 +133,42 @@ const CredentialCard: React.FC<CredentialCardProps> = ({ credential, style = {},
   }, [credential])
 
   const renderCredentialCardHeader = () => {
-    const calculatePercentageWidthWithLogo = (
-      headerDimensions: LayoutRectangle,
-      headerLogoDimensions: LayoutRectangle,
-      halved = true
-    ) => {
-      const { width: header } = headerDimensions
-      const { width: headerLogo } = headerLogoDimensions
-      const remainingHeader = header - headerLogo
-      return `${((halved ? 50 : 100) * (remainingHeader - 0.5 * paddingHorizontal)) / header}%`
-    }
-
-    const scaleCredentialName = (hideIssuer = false): string | number | undefined => {
-      return hideIssuer ? '100%' : '50%'
-    }
-
-    const scaleCredentialNameByLogo = (
-      headerDimensions: LayoutRectangle,
-      headerLogoDimensions: LayoutRectangle,
-      hideIssuer = false
-    ): string | number | undefined => {
-      return calculatePercentageWidthWithLogo(headerDimensions, headerLogoDimensions, !hideIssuer)
-    }
-
     return (
       <View
         testID={testIdWithKey('CredentialCardHeader')}
-        style={[styles.headerContainer]}
-        onLayout={({ nativeEvent: { layout } }) => setHeaderDimensions(layout)}
+        style={[styles.headerContainer, { flex: 1, alignContent: 'flex-start', alignItems: 'flex-start' }]}
```

Comment:
Quite a bit of in-line CSS. Do you think it's more inline with convention to create a `const style` like other components / screens?

Reply:
I agree. but can we refactor later? I just need to figure out the right incantation first :D